README.md

```diff
@@ -18,7 +18,18 @@ provide their own model implementations and use GraphStorm training pipeline to
 ### Installation
 GraphStorm is compatible to Python 3.8+. It requires PyTorch 1.13+, DGL 1.0+ and transformers 4.3.0+. GraphStorm only supports DGL up to version 2.3.0.
 
-GraphStorm can be installed with pip and it can be used to train GNN models in a standalone mode. To run GraphStorm in a distributed environment, we recommend users to using [Docker](https://docs.docker.com/get-started/overview/) container to reduce environment setup efforts. A guideline to setup GraphStorm running environment can be found at [here](https://graphstorm.readthedocs.io/en/latest/install/env-setup.html#setup-graphstorm-docker-environment) and a full instruction on how to setup distributed training can be found [here](https://graphstorm.readthedocs.io/en/latest/cli/model-training-inference/distributed/cluster.html).
+GraphStorm can be installed with pip and it can be used to train GNN models in a standalone mode. To run GraphStorm in a distributed environment, we recommend users to using [Docker](https://docs.docker.com/get-started/overview/) container to reduce environment setup efforts. A guideline to setup GraphStorm running environment can be found at [here](https://graphstorm.readthedocs.io/en/latest/install/env-setup.html#setup-graphstorm-docker-environment) and a full instruction on how to setup distributed training can be found [here](https://graphstorm.readthedocs.io/en/latest/cli/model-training-inference/distributed/cluster.html). For quick installation, please refer to the following command:
+```
+pip install graphstorm
```

Comment:
Can we put this last as the other libs are graphstorm requirements?

Reply:
No we cannot. Because graphstorm dependencies will automatically fetch the latest torch/dgl version. But we have to use specific version here in this PR.

Reply:
I'm not sure i understand, if we first install torch etc. and install graphstorm last the dependencies will be satisfied and torch and other covered deps won't be installed right?

Reply:
I just double check that. We can move the ``pip install graphstorm`` to the end and it works. I will update here.